src/index.html

```diff
@@ -6,9 +6,38 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Henny+Penny&family=Lexend+Deca:wght@400;500;600;700&family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <h1 class="main-header">Moyo header</h1>
+    <header class="header">
+    </header>
+    <title>Document</title>
+</head>
```

Comment:
you have 2 `</head>` closing tags in line 13 and 19 but one `<head>` opening tag in line 3 remove closing tag in line 19. it is extra